index.d.ts

```diff
@@ -79,7 +79,7 @@ declare namespace LightMyRequest {
     trailers: { [key: string]: string }
     payload: string
     body: string
-    json: () => object
+    json: () => any
```

Comment:
I was asking myself how [`@types/node`](https://github.com/DefinitelyTyped/DefinitelyTyped) has defined what `JSON.parse` returns but I didn't find anything useful.
Do you think their definition could help to reduce the `any` range?

Reply:
There's a bit of discussion on this in #82 which might be of interest